src/main/java/seedu/duke/ui/DukeUI.java

```diff
@@ -41,4 +44,32 @@ public void printIntroduction() {
     public void printMessage(String message) {
         System.out.println(message);
     }
+
+    /**
+     * Prints help messages for the "help" command.
+     */
+
+    public void printHelpMessages() {
+        System.out.println("COMMAND SUMMARY:");
+        System.out.println(DIVIDER);
+        System.out.println("Command: list");
+        System.out.println("Lists out current ingredients and their respective quantities stored in the fridge.");
+        System.out.println(DIVIDER);
+        System.out.println("Command: add <ingredient> /qty <quantity>");
+        System.out.println("Add an ingredient of specified quantity to your ingredient inventory.");
+        System.out.println(DIVIDER);
+        System.out.println("Command: remove <ingredient> /qty <quantity>");
+        System.out.println("Remove an ingredient of specified quantity from your ingredient inventory.");
+        System.out.println(DIVIDER);
+        System.out.println("Command: recipe possible");
+        System.out.println("Lists out all the recipes that can be made by the ingredients that are currently " +
+                "in the ingredient inventory.");
+        System.out.println(DIVIDER);
+        System.out.println("Command: recipe all");
+        System.out.println("Lists out all the recipes currently stored in the recipe inventory.");
+        System.out.println(DIVIDER);
+        System.out.println("Command: bye");
+        System.out.println("Exits the program");
+        System.out.println(DIVIDER);
+    }
```

Comment:
Would it be better to extract the "help" text for each command to the individual command classes themselves?

Reply:
I do agree with extracting the "help" text but maybe it would be better if it was still all done in DukeUI for ease of access since it is just the printing of commands and does not require data from the individual command classes themselves.